imperial_coldfront_plugin/settings.py

```diff
@@ -24,6 +24,22 @@
 GPFS_API_TIMEOUT = ENV.int("GPFS_API_TIMEOUT", default=300)
 """Timeout for requests that require creating, deleting or updating resources."""
 
+GPFS_FILESYSTEM_NAME = ENV.str("GPFS_API_FILESYSTEM_NAME", default="")
+"""Name of the GPFS filesystem."""
+
+GPFS_FILESET_PATH = ENV.str("GPFS_API_FILESET_PATH", default="")
+"""Path to the fileset."""
+
+GPFS_PERMISSIONS = ENV.str("GPFS_API_PERMISSIONS", default="700")
+"""Permissions for the fileset."""
+
+GPFS_FILES_QUOTA = ENV.int("GPFS_API_FILES_QUOTA", default=1000)
+"""Quota for the fileset."""
+
+GPFS_ENABLED = bool(
+    GPFS_FILESYSTEM_NAME and GPFS_FILESET_PATH and GPFS_PERMISSIONS and GPFS_FILES_QUOTA
+)
+
```

Comment:
I'd base this on whether the url, password and username settings have been provided.